kubernetes/back-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
```

Comment:
Can we add a minimal README to the `kubernetes` directory similar to what the [spring-petclinic-reactive repo has](https://github.com/pixie-io/spring-petclinic-reactive/blob/0b77c50e0b218d44414008b7f29d6ea6b60207fb/kubernetes/README.md)?